aten/src/TH/generic/THTensorEvenMoreMath.cpp

```diff
@@ -569,7 +569,10 @@ scalar_t THTensor_(minall)(THTensor *tensor)
   scalar_t theMin;
   scalar_t value;
 
-  THArgCheck(THTensor_nDimensionLegacyAll(tensor) > 0, 1, "tensor must have one dimension");
+  if (THTensor_(nElement)(tensor) == 0){
+    return std::numeric_limits<scalar_t>::max();
```

Comment:
I thought we weren't going to do this but were going to have the same behavior as the dimensional case?

Reply:
That's precisely the opposite of what I was thinking :)